### ISSUE TITLE
Interact with the siren

### ISSUE THREAD

**Issue by ricardolima-web** [MadDoct/ESP-CrowAlarmInterface#6]
Hello
I have a runner 4 using this code and we are seeing the zones being affected as supposed.
But now I would like to add more sensor (wireless sensors) and add it to home assistant and use it to trigger the siren of the original alarm. It is possible? 
I can use the relay to trigger the siren, but for that it is needed to broke the connection with the original alarm. There is any more idea?

Thank you for the interface!
Obrigado. É bom ver malta portuguesa a fazer destas coisas.

**Comment by MadDoct** [comment]
yes, it is already implemented in the code... If you send a payload of "alarme" to the command topic, it will trigger the alarm, but the panel must have that feature activated. I don't recall how to do it exactly...

**Comment by OakfieldDrive** [comment]
I am also interested in doing the same. MadDoct, any change to share you could share an example of code on how to send this payload? That would be greatly appreciated.

**Comment by ricardolima-web** [comment]
Thank you MadDoct 
It is working. 
Basically I have tested it by using the Mqttbox with the following info
![imagem](https://github.com/user-attachments/assets/1c043028-8887-4eea-84e4-2c1a9507753e)
And the alarm just started working (and the siren making noise)
